src/app/intelligence.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import {URLSearchParams, Http, Jsonp} from "@angular/http";
+import {Store} from "@ngrx/store";
+import * as fromRoot from './reducers';
+import {Observable} from "rxjs";
+
+@Injectable()
+export class IntelligenceService {
+  server = 'http://api.susi.ai';
+  searchURL = 'http://' + this.server + '/susi/chat.json';
+  constructor(private http: Http, private jsonp: Jsonp, private store: Store<fromRoot.State>) {
+  }
+  getintelligentresponse(searchquery) {
+    let params = new URLSearchParams();
+    params.set('q', searchquery);
+    // params.set('QueryClass', 'MaxHits=5');
+
```

Comment:
Remove the comment also. There is no need to add `QueryClass` and `MaxHits` in future. We won't be using them.